internal/server/tests.go

```diff
@@ -2,25 +2,27 @@ package server
 
 import (
 	"context"
-	"github.com/quan-to/chevron/internal/config"
-	"github.com/quan-to/chevron/internal/database"
 	"github.com/quan-to/chevron/internal/tools"
 	"github.com/quan-to/chevron/internal/vaultManager"
 	"github.com/quan-to/chevron/pkg/models"
 	"net/http"
 
 	"github.com/gorilla/mux"
 	"github.com/quan-to/slog"
-	r "gopkg.in/rethinkdb/rethinkdb-go.v6"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).